tests/parser/features/iteration/test_range_in.py

```diff
@@ -71,4 +71,33 @@ def testin() -> bool:
     return False
     """
     with pytest.raises(TypeMismatchException):
-        c = get_contract(code)
+        get_contract(code)
+
+
+def test_ownership():
+    code = """
+
+owners: address[10]
+
+def __init__():
+    self.owners[0] = msg.sender
+
+def set_owner(i: num, new_owner: address):
+    assert msg.sender in self.owners
+    self.owners[i] = new_owner
+
+def is_owner() -> bool:
+    return msg.sender in self.owners
+    """
+
+    c = get_contract(code)
+
+    assert c.is_owner() is True  # contract creator is owner.
+    assert c.is_owner(sender=t.k1) is False  # no one else is.
+
+    # only an owner may set another owner.
+    with pytest.raises(t.TransactionFailed):
+        c.set_owner(1, t.a1, sender=t.k1)
+
+    assert c.set_owner(1, t.a1) is None
```

Comment:
Unnecessary to write it this way, just change to `c.set_owner(1, t.a1)`

Reply:
:+1: